packages/admin-page/src/page.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * An admin page wrapper
+ *
+ * @package jetpack-admin-page
+ */
+
+namespace Automattic\Jetpack\AdminPage;
+
+/**
+ * Create and render a Jetpack logo.
+ */
+class Page {
+	/**
+	 * Absolute URL of the Jetpack logo.
+	 *
+	 * @var string
+	 */
+	private $header;
+
+	private $footer;
+
+	/**
+	 * Constructor.
+	 * You can optionally pass a URL to override the default one.
+	 */
+	public function __construct( $page_hook = null ) {
+		if ( $page_hook ) {
+			if ( is_array( $page_hook ) ) {
+				foreach ( $page_hook as $hook ) {
+					add_action( "admin_print_styles-$hook", array( $this, 'admin_styles' ) );
+				}
+			} else {
+				add_action( "admin_print_styles-$page_hook", array( $this, 'admin_styles' ) );
+			}
+		}
+	}
+
+	public function admin_styles() {
+		// TODO: implement min and rtl styles in some sort of build process?
+		$version = time(); // where does this come from?
```

Comment:
Could we use the `filemtime()` of the stylesheet?

Reply:
I wonder if this is am optimisation thing. Outputting from `time()` may be much faster than reading `filemtime()`.

Reply:
The reason to have it is to force caching it in the browser, and to have a mechanism to force cleaning of that cache if we need. That's why a `filemtime` here is convenient, but I agree on it being slower. `time()` there though will be different every time, which means we won't be getting the benefit of browser caching for the corresponding asset.